### ISSUE TITLE
[10.0][11.0] account_bank_statement_import_camt_oca: Please add tag AddtlInf to the name field

### ISSUE THREAD

**Issue by yung-wang** [OCA/bank-statement-import#187]
Impacted versions: [10.0][11.0]
Module: account_bank_statement_import_camt_oca

When the SDD is not successful (because not enough balance) the bank will send some information in the XML tag AddtlInf when you import the XML CAMT file. In the module this information is not written anywhere. It would be nice to add the information in the name field of Odoo when you import the bankstatement.

```
<NtryDtls>
<TxDtls>
<Refs>
<EndToEndId>DLDLASF-SDD-75114</EndToEndId>
<MndtId>PC.CBA.4897</MndtId>
</Refs>
<BkTxCd>
<Prtry>
<Cd>108</Cd>
</Prtry>
</BkTxCd>
<RltdPties>
<Dbtr>
<Nm>COMPANY BV</Nm>
<PstlAdr>
<AdrTp>ADDR</AdrTp>
<Ctry>NL</Ctry>
<AdrLine>STRAAT 2-6</AdrLine>
<AdrLine>3500 AA UTRECHT</AdrLine>
</PstlAdr>
</Dbtr>
<DbtrAcct>
<Id>
<IBAN>NL78ABNA0123456789</IBAN>
</Id>
</DbtrAcct>
</RltdPties>
<RltdAgts>
<DbtrAgt>
<FinInstnId>
<BIC>ABNANL2A</BIC>
</FinInstnId>
</DbtrAgt>
</RltdAgts>
<RmtInf>
<Ustrd>
Information for the customer
</Ustrd>
</RmtInf>
<RtrInf>
<Rsn>
<Cd>AM04</Cd>
</Rsn>
<AddtlInf>Not enough balance</AddtlInf>
</RtrInf>
<AddtlTxInf>/CSID/NL90ZZZ012345678912</AddtlTxInf>
</TxDtls>
</NtryDtls>
```

The easiest way would change the parser.py and change the line from
`'./ns:RmtInf/ns:Ustrd'`
to
`'./ns:RmtInf/ns:Ustrd|./ns:RtrInf/ns:AddtlInf',`
The information will be written after the Ustrd. Can you have a look if there is a better solution?

**Comment by pedrobaeza** [comment]
How is SEPA related to this repository?

**Comment by yung-wang** [comment]
Sorry forgot to say that when you import the XML CAMT file, the bank will sent extra information in the tag AddtlInf. Do you mean this?

**Comment by pedrobaeza** [comment]
Yes, it's very important to specify which module are you talking about in the title, and also what is your exact proposal.

**Comment by yung-wang** [comment]
Its better now?

**Comment by pedrobaeza** [comment]
OK, I still don't understand what to do, but this is because I don't know CAMT specification. I suppose others familiar with the module will understand it.

**Comment by astirpe** [comment]
This issue can be closed: fixed in 10.0, 11.0 and 12.0.
@pedrobaeza
